CombineTools/scripts/plot1DScan.py

```diff
@@ -32,11 +32,11 @@ def Eval(obj, x, params):
     return obj.Eval(x[0])
 
 
-def BuildScan(scan, param, files, color, yvals, ycut):
+def BuildScan(scan, param, files, color, yvals, ycut, debug=False):
     graph = read(scan, param, files, ycut)
```

Comment:
Is a debug option really needed? I don't see any gain for debugging purposes, at least in the current state of the printouts.